package.json

```diff
@@ -31,6 +31,7 @@
     "lodash.mergewith": "^4.6.0",
     "postcss": "^6.0.14",
     "srcset": "^1.0.0",
+    "url": "^0.11.0",
```

Comment:
url is a built-in node module and should not be an npm dependency. Make sure this is all being tested against the built in url module and not whatever this might be...

Reply:
ah yes, made this change.